clang-tools-extra/clang-doc/Representation.h

```diff
@@ -431,6 +431,8 @@ struct EnumValueInfo {
   // Stores the user-supplied initialization expression for this enumeration
   // constant. This will be empty for implicit enumeration values.
   SmallString<16> ValueExpr;
+
+  std::vector<CommentInfo> Description; // Comment description of this field.
```

Comment:
Shouldn't this be `///`? Otherwise I'd make it something like `// Not a documentation comment` or `// A comment that shouldn't show up in documentation`.